src/main/java/org/opentripplanner/gtfs/mapping/LocationMapper.java

```diff
@@ -31,19 +32,20 @@ FlexStopLocation map(org.onebusaway.gtfs.model.Location orginal) {
 
   private FlexStopLocation doMap(org.onebusaway.gtfs.model.Location gtfsLocation) {
     var name = NonLocalizedString.ofNullable(gtfsLocation.getName());
-    FlexStopLocation otpLocation = new FlexStopLocation(mapAgencyAndId(gtfsLocation.getId()), name);
-
-    otpLocation.setUrl(NonLocalizedString.ofNullable(gtfsLocation.getUrl()));
-    otpLocation.setDescription(NonLocalizedString.ofNullable(gtfsLocation.getDescription()));
-    otpLocation.setZoneId(gtfsLocation.getZoneId());
+    Geometry geometry = null;
     try {
-      otpLocation.setGeometry(
-        GeometryUtils.convertGeoJsonToJtsGeometry(gtfsLocation.getGeometry())
-      );
+      geometry = GeometryUtils.convertGeoJsonToJtsGeometry(gtfsLocation.getGeometry());
     } catch (UnsupportedGeometryException e) {
-      LOG.warn("Unsupported geometry type for " + gtfsLocation.getId());
+      LOG.warn("Unsupported geometry type for {}", gtfsLocation.getId());
```

Comment:
I think it would be better to error in this case. That way we always have an Geometry for the `StopLocation`s

Reply:
increased log level to error